.github/workflows/cibuild.yml

```diff
@@ -12,14 +12,14 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-16.04, ubuntu-18.04]
+        os: [ubuntu-18.04, ubuntu-20.04]
```

Comment:
Is there a particularly compelling reason to test on multiple Ubuntu versions instead of `ubuntu-latest`?

Reply:
Good question I will defer to @zrax, but I know he generally seems to want stuff to work all the way back to the previous ubuntu LTS.